classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+    def initialize(label)
+      @label = label
+      @students = []
+    end
+
+    attr_accessor :label, :students
```

Comment:
- [ ] Kindly set the students to be an `attr_reader` instead of an `attr_accessor` as we want to be able to only read the students for a classroom. The `add_student` method handles the writing of new students so an `attr_accessor` for students is not necessary 👍